Cargo.toml

```diff
@@ -27,7 +27,7 @@ features = ["nightly", "numpy"]
 no-std-compat = { version = "0.4.1", default-features = false, features = [ "alloc", "compat_hash" ] }
 rand = { version = "0.8.5", default-features = false, features = ["std_rng"] }
 rand_distr = { version = "0.4.3", default-features = false, features = ["std_math"] }
-matrixmultiply = { version = "0.3.2", default-features = false }
+matrixmultiply = { version = "0.3.2", default-features = false, features = ["threading"] }
```

Comment:
Since threading feature requires std, let's add this to the std feature in the section below instead (see example I posted in issue)